components/TravelStatusPicker.js

```diff
@@ -27,18 +30,37 @@ const TravelStatusPicker = () => {
     setIsModalVisible(false);
   };
 
+const renderTravelButton = (buttonText = '') => {
+  return (
+    <TouchableOpacity
+      style={styles.button}
+      onPress={openModal}
+    >
+      <Text style={styles.buttonText}>{buttonText}</Text>
+    </TouchableOpacity>
+  );
+}
+
+// Refactored
+const renderTravel = (hasSetTravel = false) => {
+  return hasSetTravel
+  ? renderTravelButton(travelStatus)
+  : renderTravelButton(setTravelStatus(APP_STRINGS.no))
```

Comment:
do this `setTravelStatus(APP_STRINGS.no` before the ternary too..? Let's talk about these

Reply:
Hmm. Okay, I'm way down to chat about if you got some time. Not sure what you mean here...